src/components/IssueBlock/index.tsx

```diff
@@ -64,7 +63,7 @@ const IssueBlock = ({ issue }: IssueBlockProps) => {
           </div>
         </div>
         <div className="ml-2 flex flex-shrink-0 flex-wrap">
-          <Link href={`/issues/${issue.id}`} passHref>
+          <Link href={`/issues/${issue.id}`} passHref legacyBehavior>
```

Comment:
@gauthier-th why was legacyBehavior prop added to this when `<a>` has been removed?

> An <a> element is no longer required as a child of <Link>. Add the legacyBehavior prop to use the legacy behavior or remove the <a> to upgrade.

Reply:
See the first child, Button? It is a `<a>`, so we need to keep the legacy behavior